<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Mesa">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-230</x>
      <y>-30</y>
      <width>1281</width>
      <height>701</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="marco">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>140</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="marcos">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>201</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Palo</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="boton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>380</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Generar Carta</string>
    </property>
   </widget>
   <widget class="QLabel" name="Carta1">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>480</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>1.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Carta2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>480</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>2.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Carta3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>3.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Carta4">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>480</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>4.0</string>
    </property>
   </widget>
   <widget class="QLabel" name="Carta5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>480</y>
      <width>121</width>
      <height>181</height>
     </rect>
    </property>
    <property name="text">
     <string>5.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
